docs/tutorial.md

```diff
@@ -0,0 +1,120 @@
+# urdf2webots Tutorial
+
+## Prerequisites:
+Make sure you have the newest version of urdf2webots installed. If you have an older version installed via pip, use this command:
+
+```
+pip install --no-cache-dir --upgrade urdf2webots
+```
+
+Alternatively, if you want to install from source, follow the instructions in the [Main README](../README.md)
+<br /> 
+<br /> 
+## Creating the URDF from git repo (optional if urdf already available)
+
+Example using this KUKA ros repo:
+https://github.com/ros-industrial/kuka_experimental
+
+```
+cd catkin_ws/src
+git clone https://github.com/ros-industrial/kuka_experimental
+cd ..
+catkin_make  # or catkin build  depending on what you use
+source devel/setup.bash
+```
+**Note:** If you want to convert this exact same robot, you will have to replace the file at
+kuka_experimental/kuka_lbr_iiwa_support/meshes/lbr_iiwa_14_r820/visual/base_link.dae
+With this:
+[base_link.dae](https://drive.google.com/file/d/1J0dVuDOW7k3wa6Gj0vpjKzlNMzQHOAfD/view?usp=sharing)
+<br /> 
+
+Navigate to the launch folder and open the launch file that launches your robot or displays it in Rviz. For this tutorial we chose the KUKA lbr iiwa robot. The launch file is:
+`/kuka_experimental/kuka_lbr_iiwa_support/launch/load_lbr_iiwa_14_r820.launch`
+Opening the launch file we look for the line, uploading the robot_description parameter, here it is this line:
```

Comment:
```suggestion
Opening the launch file we look for the line, uploading the robot_description parameter, here is this line:
```

Reply:
The 'it' is correct.   here it (the thing we are looking for) is this line: